src/Compilers/CSharp/Portable/Symbols/Tuples/TupleErrorFieldSymbol.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// Represents a field of a tuple type (such as (int, byte).Item1)
+    /// that doesn't have a corresponding backing field within the tuple underlying type.
+    /// Created in response to an error condition.
```

Comment:
That is because you think we live in the perfect world. :-) However, we should be prepared that ValueTuple\<,\> doesn't have the Item1 field of the expected type, i.e. malformed. The comment specifically talks about an error situation. If you think it is not clear enough, please provide your wording and I will paste it in.